xla/service/gpu/fusions/triton.cc

```diff
@@ -114,6 +114,9 @@ absl::StatusOr<FusionEmissionResult> TritonFusion::Emit(
         triton_config.set_block_k(64);
         triton_config.set_block_n(64);
         triton_config.set_split_k(1);
+        triton_config.set_num_stages(1);
```

Comment:
Is this necessary? I don't think `num_stages`, `num_warps`, and `num_ctas` are read from anywhere else than `block_level_parameters`---are they?

Reply:
If value is not set here it fails following check:
https://github.com/openxla/xla/blob/8caa27b40e443672f2eac42a06cdbe7680547f16/xla/service/gpu/matmul_utils.cc#L824